src/index.js

```diff
@@ -22,7 +22,7 @@ export const tall = (url, options) => {
     return request({ method, protocol, host, path }, response => {
       if (response.headers.location && opt.maxRedirects) {
         opt.maxRedirects--
-        return resolve(tall(response.headers.location, opt))
+        return resolve(tall(response.headers.location.startsWith('http')?response.headers.location:`${protocol}//${host}${response.headers.location}`,opt))
```

Comment:
Being extra picky here, but what if the header contains something like: `http/website/index.html` ? Maybe we should check for `http://` and `https://` entirely.

Reply:
There is allready code from you in the first part of tall to check for invalid URLs - no need to replicate.
The use case to cover is the url with an absolute path which is not allowed by definition but quit common.